custom_components/opensprinkler/__init__.py

```diff
@@ -367,6 +367,14 @@ def state(self):
         return self._state
 
 
+class OpenSprinklerControllerEntity:
+    def stop(self):
+        """Stops all stations."""
+        for _, station in self._controller.stations.items():
```

Comment:
I think we should use this instead: `controller.stop_all_stations()` Although maybe we should rename that method and/or the service. Basically it stops any running stations *and* any stations that are `waiting`.